lib/compiler.js

```diff
@@ -1,9 +1,13 @@
 var hasOwn = Object.prototype.hasOwnProperty;
 var assert = require("assert");
-var types = require("ast-types");
+var types = require("ast-types/fork")([
+  require("ast-types/def/esprima"),
+  require("ast-types/def/babel6")
+]);
```

Comment:
As we chatted about yesterday, the longer term plan is to avoid using the `ast-types` library if we can get away with something lighter-weight.